packages/fetch-http-handler/src/fetch-http-handler.ts

```diff
@@ -21,6 +21,10 @@ export interface BrowserHttpOptions {
    * ReadableStream.
    * This option is only useful in ReactNative.
    */
+  // TODO can we get rid of this? is there a scenario where a client wants to
+  //  insist on a Blob, even if stream is implemented? regardless, comment above
+  //  is outdated, this is not only useful in ReactNative due to variety of fetch
+  //  implementations
```

Comment:
I think the same way. If we rely on dynamicly detecting the `ReadableStream` implementation in the environment, we should remove it.